src/components/configure/ExportImport.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { Upload, message, Form, Select, Input, Button } from 'antd';
+import { Icon } from 'antd';
+import { useSelector, useDispatch } from 'react-redux';
+import { setProjectConfig, getProjectConfig, notify } from '../../utils';
+import { useParams } from "react-router-dom";
+import { get, set, increment, decrement } from 'automate-redux';
+import client from "../../client"
+
+
+const YAML = require('yamljs');
+const yaml = require('js-yaml');
+
+const ExportImport = ({ object }) => {
+    const { projectID, selectedDB } = useParams()
+    const [Type, setType] = useState('');
+    const dispatch = useDispatch()
+
+
+    const projects = useSelector(state => state.projects)
+    const projectName = getProjectConfig(projects, projectID, "name")
+
+    const downLoadJson = e => {
+        e.preventDefault();
+
+        const data = new Blob([JSON.stringify(object)], { type: 'content-type/json' });
+        const jsonURL = window.URL.createObjectURL(data);
+        const jsonLink = document.createElement('a');
+        jsonLink.href = jsonURL;
+        jsonLink.setAttribute('download', 'config.json');
+        jsonLink.click();
+
+    }
+    const downLoadYaml = e => {
+        e.preventDefault();
+        const nativeObject = YAML.parse(JSON.stringify(object));
+        const yamlString = YAML.stringify(nativeObject, 4);
+        const data = new Blob([yamlString], { type: 'text/yaml' });
+        const yamlURL = window.URL.createObjectURL(data);
+        const yamlLink = document.createElement('a');
+        yamlLink.href = yamlURL;
+        yamlLink.setAttribute('download', 'config.yaml');
```

Comment:
Let's keep the name of the file that is being downloaded as its project-id rather than config